tests/acceptance/features/webUIRestrictSharing/disableSharing.feature

```diff
@@ -11,3 +11,34 @@ Feature: disable sharing
     Given the setting "shareapi_enabled" of app "core" has been set to "no"
     When user "user1" logs in using the webUI
     Then it should not be possible to share folder "simple-folder" using the webUI
+
+  Scenario: Users tries to share from favorites page when sharing is disabled
+    Given the setting "shareapi_enabled" of app "core" has been set to "no"
+    And user "user1" has favorited element "lorem.txt"
+    And user "user1" has favorited element "simple-folder"
+    And user "user1" has logged in using the webUI
+    When the user browses to the favorites page
+    Then it should not be possible to share folder "simple-folder" using the webUI
+    And it should not be possible to share file "lorem.txt" using the webUI
+
+  Scenario: Check file presence from shared-with-me page when sharing is disabled
```

Comment:
Can you please verify this is the correct expected behavior and raise an issue if not. Compare with core for reference.